frontend/src/components/AddNewTripComponents/AddNewTrip.tsx

```diff
@@ -244,10 +250,10 @@ export default function AddNewTrip({addNewTrip}: NewTripProps) {
                                              setCustomActivityItem={setCustomActivityItem}
                                              amountOfCustomActivityItem={amountOfCustomActivityItem}
                                              setCustomActivityItemEmission={setCustomActivityItemEmission}/>
-                        </form>
+                        </div>
```

Comment:
Ok, Du hast ja in den ganzen switch-cases einen wrapper Tag um die React-Komponente, der vorher ein Form-Tag war und den Du nun zu Div geändert hast. Aber ich frage mich, ob Du diese div um den Komponentenaufruf überhaupt benötigst oder ob es genügen würde die Komponente ohne Wrapper aufzurufen. (Betrifft nicht diese Zeile, sondern insgesamt diese Switch)